locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -145,6 +145,15 @@ EXPOSE 8080
 CMD [ "npm", "start" ]
 ```
 
+## .dockerignore file
+
+Prevent your local modules and debug log from being copied into your Docker image:
```

Comment:
Maybe just make this statement a little clearer, something like:
To prevent your local modules and debug logs from being copied onto your Docker image, create a `.dockerignore` file with the below contents:

Also, you should mention where this file will go.